app/src/main/java/app/passwordstore/ui/crypto/DecryptActivity.kt

```diff
@@ -237,8 +243,6 @@ class DecryptActivity : BasePGPActivity() {
     val outputStream = ByteArrayOutputStream()
     val result = repository.decrypt(passphrase, identifiers, message, outputStream)
     if (result.isOk) {
-      if (features.isEnabled(EnablePGPPassphraseCache))
-        settings.edit { putBoolean(PreferenceKeys.CLEAR_PASSPHRASE_CACHE, clearCache) }
```

Comment:
Why are we changing this?

Reply:
I thought I move `settings.edit { ... }` to the place where the passphrase is written to the cache (now on line 226). Both operations belong together somehow. A change of this setting is only persisted if the passphrase cache could be successfully written too (they are in the same `runCatching` block). Its now at the same place in both, `DecryptActivity` and `AutofillDycryptActivity`.